gguf-py/examples/reader.py

```diff
@@ -18,28 +20,28 @@ def read_gguf_file(gguf_file_path):
     reader = GGUFReader(gguf_file_path)
 
     # List all key-value pairs in a columnized format
-    print("Key-Value Pairs:")
+    logger.info("Key-Value Pairs:")
     max_key_length = max(len(key) for key in reader.fields.keys())
     for key, field in reader.fields.items():
         value = field.parts[field.data[0]]
-        print(f"{key:{max_key_length}} : {value}")
-    print("----")
+        logger.info(f"{key:{max_key_length}} : {value}")
+    logger.info("----")
 
     # List all tensors
-    print("Tensors:")
+    logger.info("Tensors:")
     tensor_info_format = "{:<30} | Shape: {:<15} | Size: {:<12} | Quantization: {}"
-    print(tensor_info_format.format("Tensor Name", "Shape", "Size", "Quantization"))
-    print("-" * 80)
+    logger.info(tensor_info_format.format("Tensor Name", "Shape", "Size", "Quantization"))
+    logger.info("-" * 80)
     for tensor in reader.tensors:
         shape_str = "x".join(map(str, tensor.shape))
         size_str = str(tensor.n_elements)
         quantization_str = tensor.tensor_type.name
-        print(tensor_info_format.format(tensor.name, shape_str, size_str, quantization_str))
+        logger.info(tensor_info_format.format(tensor.name, shape_str, size_str, quantization_str))
```

Comment:
All of this should be written to stdout (e.g. with `print()`), as it is the result of the program.